clients/src/test/java/org/apache/kafka/clients/admin/internals/DeleteRecordsHandlerTest.java

```diff
@@ -63,11 +74,11 @@ public class DeleteRecordsHandlerTest {
     @Test
     public void testBuildRequestSimple() {
         DeleteRecordsHandler handler = new DeleteRecordsHandler(recordsToDelete, logContext, timeout);
-        DeleteRecordsRequest request = handler.buildBatchedRequest(node.id(), mkSet(t0p0, t0p1)).build();
+        DeleteRecordsRequest request = handler.buildBatchedRequest(node1.id(), mkSet(t0p0, t0p1)).build();
         List<DeleteRecordsRequestData.DeleteRecordsTopic> topicPartitions = request.data().topics();
```

Comment:
While we're fixing this test, can we also rename this variable. Something like `topics` would fit better I think.